File: lib_book_common/src/main/java/com/ebook/common/BaseApplication.java
package com.ebook.common;

import com.ebook.common.util.log.KLog;

public class BaseApplication extends com.xrn1997.common.BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        KLog.init(BuildConfig.IS_DEBUG);
    }
}

File: lib_book_common/src/main/java/com/ebook/common/BaseApplication.kt
package com.ebook.common

import com.ebook.common.util.log.klog.KLog
import com.xrn1997.common.BaseApplication

open class BaseApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        KLog.init(BuildConfig.IS_DEBUG)
    }
}
